tests/platforms/test_iguazio.py

```diff
@@ -227,3 +229,23 @@ def test_is_iguazio_session_cookie():
         is True
     )
     assert mlrun.platforms.is_iguazio_session_cookie("dummy") is False
+
+
+def test_min_iguazio_version():
+    mlconf.igz_version = "3.5.4"
+
+    for case in [
+        ["3.5.5"],
+        ["3.6.0"],
+        ["3.5.5-b25.20231224135202"],
+        ["4.0.0"],
+        ["3.2.0", "3.6.0"],
+    ]:
+        print(f"Testing case: {case}")
```

Comment:
change print to log and use pytest parameterize because it will continue for the other cases if one of them fails